src/core.ts

```diff
@@ -78,6 +78,15 @@ export enum Html5QrcodeScanType {
     SCAN_TYPE_FILE = 1      // File based scanner.
 }
 
+/**
+ * Types of scans selected by the consumer
+ */
+ export enum Html5QrcodeSelectedScanType {
```

Comment:
Could we use the existing definition? i.e. `Html5QrcodeScanType`

devs can pass support as 

```js
supportedTypes: Array<Html5QrcodeScanType> = [SCAN_TYPE_CAMERA, SCAN_TYPE_FILE];
```

This will be more scalable to more additions